### PR TITLE
 Check for $TAG_TEMPLATE var on building GitHub source url

### PR DESCRIPTION
**Pull request by igortg**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
This needs news

File: tests/conftest.py [hunk 1]
```diff
@@ -38,3 +39,22 @@ def gitecho(request):
     aliases['git'] = lambda args: 'Would have run: ' + ' '.join(args) + '\n'
     yield None
     del aliases['git']
+
+
+def rmtree(dirname):
```
Comment (CJ-Wright) [inline tests/conftest.py]: We didn't have this? Nice catch.

Comment (scopatz) [reply tests/conftest.py]: I think that there is a rmtree in distutils in the standard library

Comment (igortg) [reply tests/conftest.py]: Just checked, same problem with remove_tree() from distutils (it's like calling rmtree with `ignore_errors=True`)

Comment (scopatz) [reply tests/conftest.py]: Ahh fair.  Sorry I didn't read the whole docstring.

**Comment by scopatz** [comment]
Thanks @igortg! Please just add a news entry as per https://regro.github.io/rever-docs/devguide.html#changelog

**Comment by igortg** [comment]
Hey guys,

There is something missing on this PR so it could be merged? Any problems with the implementation, please let me knowç

**Comment by scopatz** [comment]
Whoops sorry! In the future, please ping the issue/PR once you think it is ready again, so that we get an email :wink:

**Pull request merged**